scrunch/version.py

```diff
@@ -1,4 +1,9 @@
-from importlib_metadata import version
+try:
+    # py <= 3.7
+    from importlib_metadata import version
+except ModuleNotFoundError:
+    # py >= 3.8
+    from importlib.metadata import version
```

Comment:
This is the exact same import?

Reply:
in py<3.8, there was the package with the underscore. Now we have importlib.metadata. We did not notice the issue because we were just running tests for py3.6 and 2.7, but I wanted to add 3.11 and I got the error.